FWCore/Framework/interface/WorkerRegistry.h

```diff
@@ -37,7 +38,7 @@ namespace edm {
 
   class WorkerRegistry {
   public:
-    explicit WorkerRegistry(std::shared_ptr<ActivityRegistry> areg);
+    explicit WorkerRegistry(std::shared_ptr<ActivityRegistry> areg, ModuleTypeResolverMaker const* resolverMaker);
     WorkerRegistry(std::shared_ptr<ActivityRegistry> areg, std::shared_ptr<ModuleRegistry> iModReg);
```

Comment:
Why doesn't this one need `ModuleTypeResolverMaker const*`?

[Answer my own question] Because only the ModuleRegistry needs it and that is being passed in.